packages/insomnia-app/app/network/__tests__/network.test.ts

```diff
@@ -604,7 +605,7 @@ describe('actuallySend()', () => {
         NOPROGRESS: true,
         PROXY: '',
         TIMEOUT_MS: 0,
-        NETRC: 'Required',
+        NETRC: 2,
```

Comment:
asking for a friend... could you explain the reason for this change please?

Reply:
network.test.ts implicitly mocks node-libcurl with a handwritten and out of date file in `__mocks__/@getinsomnia/node-libcurl` as a string. I tested httpversion with a string and it doesn't work but enum does. Seems like a case of the test covering the mock and not the code under test.

Reply:
why not just directly import https://github.com/JCMais/node-libcurl/blob/develop/lib/enum/CurlNetrc.ts#L30 then?

```suggestion
        NETRC: CurlNetrc.Required,
```